### ISSUE TITLE
UI/UX: Consider a bit (few pixels) of extra spacing between the individual layer control widgets

### ISSUE THREAD

**Issue by psobolewskiPhD** [napari/napari#7471]
## 🧰 Task
The layer controls panel has always felt a bit cramped to me.
I think a few extra pixels -- literally 1 2 maybe 3 -- between the widgets would help.
For an example, see:
https://github.com/napari/napari/pull/7355#discussion_r1894544855

A bit of extra breathing room goes a long way, e.g.
`        self.button_grid.setSpacing(6)`

| With the change here to 6: | With the previous, 4: |
|----------------------------|----------------------|
| <img width="286" alt="image" src="https://github.com/user-attachments/assets/b7c60d5d-6feb-4445-9bc2-18de78704d51" /> | <img width="282" alt="image" src="https://github.com/user-attachments/assets/b3c0106c-42ca-4f37-8040-f208e2d51a1d" /> |

Heck, i'd do even a bit more...

_Originally posted by @psobolewskiPhD in https://github.com/napari/napari/pull/7355#discussion_r1894544855_

**Comment by jni** [comment]
I think the general UX advice is "big buttons with small spacing", so it might be better to use the 2 extra pixels to embiggen the buttons?

**Comment by psobolewskiPhD** [comment]
Bigger buttons would make target areas larger, so yes that would help! 
I bet the default type size is too small for a lot of people too -- it would increase the size of some of the elements:
<img width="332" alt="image" src="https://github.com/user-attachments/assets/c26f0ec5-a465-4120-8c57-b1b6a4f9f5f9" />
Note how with the larger widgets, to me, it's begging for more space between them. Visually it looks really cramped.
I'm not sure a bit of spacing would make it easier to navigate, but it would make it visually more pleasing to look at.
Bigger elements tend to need more room between them?

**Comment by DragaDoncila** [comment]
I don't think that screenshot looks cramped at all lol, but I'm happy to be overridden!

**Comment by TimMonko** [comment]
Bigger buttons! I understand the feeling about the elements maybe being too close, but I don't mind either way. 

I'm glad font size can change in preferences, BUT generally I've noticed scaling of UI is problematic! It's nice that changing the scale of my overall desktop (in Windows settings) changes the scale of napari, but it snaps between 1.0 scale and 2.0 scale. Would love something like 1.25 scale to match this AND for it to be settable in preferences. Are things like [this comment in #1710](https://github.com/napari/napari/issues/1710#issuecomment-712489853) still mysterious? or are things settable now? I'd guess not if #4290 is still open.

**Comment by psobolewskiPhD** [comment]
@TimMonko I don't think we have anyone active on the dev team using Windows, so it's hard to tell where those issues stand. 
You can experiment with `QT_SCALE_FACTOR=2 napari` using different values...

**Comment by TimMonko** [comment]
@psobolewskiPhD `QT_SCALE_FACTOR=2 napari` must not be the command in Windows - at least, I can't get it to work from the terminal.

That said, this little snippet does work:
```
import os
import napari

os.environ['QT_SCALE_FACTOR'] = '1.25'

viewer = napari.Viewer()
napari.run()
```
It scales my 1.0 screen to 1.5 and my 2.0 screen to 3.0 (it looks a bit silly). So, it's still inheriting from the `devicePixelRatio()`.
Ultimately, I'd like to be able to change each screen separately, but at least this is a start for me to think about.
